src/test/java/fm/last/moji/integration/AbstractMojiIT.java

```diff
@@ -133,4 +133,62 @@ String newKey() {
     return newKey(RandomStringUtils.randomAlphanumeric(16));
   }
 
+  private void initMogileTestData() throws IOException {
+    if(initMogileTestDataIsDone) {
+      return;
+    }
+    clearTestData();
+    uploadFile("fileOfKnownSize", "src/test/data/fileOfKnownSize.dat");
+    uploadFile("attributes", "src/test/data/fileOfKnownSize.dat");
+    uploadFile("mogileFileCopyToFile", "src/test/data/mogileFileCopyToFile.dat");
+    uploadNewRandomFile("overwriteThenReadBack");
+    uploadNewRandomFile("exists");
+    uploadNewRandomFile("notExistsAfterDelete");
+    uploadNewRandomFile("rename");
+    uploadNewRandomFile("renameExistingKey1");
+    uploadNewRandomFile("renameExistingKey2");
+    uploadNewRandomFile("updateStorageClass");
+    uploadNewRandomFile("updateStorageClassToUnknown");
+    uploadNewRandomFile("list1");
+    uploadNewRandomFile("list2");
+    uploadNewRandomFile("list3");
+    uploadNewRandomFile("getPaths");
+    initMogileTestDataIsDone = true;
+  }
+
+  private void clearTestData() throws IOException {
+    List<MojiFile> files = moji.list(keyPrefix);
+    for (MojiFile file : files) {
+      System.out.println("Deleting: " + file.getKey() + " ...");
+      file.delete();
+    }
+  }
+
+  private void uploadNewRandomFile(String key) throws IOException {
+    MojiFile file = moji.getFile(keyPrefix + key, storageClassA);
+    InputStream is = new ByteArrayInputStream(UUID.randomUUID().toString().getBytes());
+    int len;
+    byte[] data = new byte[4096];
+    OutputStream os = null;
```

Comment:
Wouldn't it be easier and less code to use something like common-io's IOUtils.copy() methods to do this? (we already have a dependency on it)